modules/microshift-examples-of-usecase.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-examples-of-usecase_{context}"]
+= Use cases for resource manifest deletion
+
+The following explains the use case in which the resource manifest deletion is used.
+
+[id="data-removal-scenario_{context}"]
+== Manifest removal scenario
+Use the following procedure in the data removal scenario to completely delete the resource defined in the kustomize manifests.
+
+.Procedure
+
+. Identify the manifest that needs to be placed in the `delete` subdirectories.
+. Create the `delete` subdirectory in which the manifest will be placed by running the following command:
++
+[source,terminal]
+----
+sudo mkdir -p _<path_of_deletefile>_ <1>
+----
+<1> Replace `_<path_of_deletefile>_` with the path of delete subdirectory such as `/usr/lib/microshift/manifests.d/delete`.
+. Move the manifest file into one of the `delete` subdirectories  under the configured manifests directory by running the following command:
++
+[source,terminal]
+----
+[sudo] mv _<path_of_manifests>_ _<path_of_deletefile>_ <1>
+----
+<1> Replace `_<path_of_manifests>_` with the path of manifest that has to be deleted such as `/usr/lib/microshift/manifests.d/some-user-manifest` and replace `_<path_of_deletefile>_` with the path of delete subdirectory such as `/usr/lib/microshift/manifests.d/delete`.
+. {microshift-short} detects and removes the resource after the manifest file is placed in the `delete` subdirectories.
+. Restart {microshift-short} by running the following command:
++
+[source,terminal]
+----
+$ sudo systemctl restart microshift
+----
+
+
+[id="upgrade-scenario_{context}"]
+== Manifest upgrade scenario
+Use the following procedure in the upgrade scenario where some resources need to be removed while retaining others to preserve data.
+
+.Procedure
+
+. Identify the manifest that requires updating.
+. Create new manifests for resource deletion and place it in the `delete` subdirectories path. It is not necessary to include the `spec` in these manifests.
+. Create new manifests that needs to be applied in the manifests path.
+. {microshift-short} detects and removes the specified object without affecting related data.
+. Restart {microshift-short} by running the following command:
```

Comment:
Same here, I think we should merge them or swap them

Reply:
done